tests/PHPUnit/HttpMockPHPUnitIntegrationTest.php

```diff
@@ -103,7 +103,7 @@ public function testErrorLogOutput()
         try {
             $this->tearDown();
             $this->fail('Exception expected');
-        } catch (ExpectationFailedException $e) {
+        } catch (\PHPUnit_Framework_ExpectationFailedException $e) {
```

Comment:
This exception class is not available in PHPUnit 6, is it?

Reply:
Yep, that's correct, but there isn't a compatibility layer on PHPUnit 5.7 for that, so the only option is to fall back to `\Exception`.

It turned out that we were running on phpunit 5.7 also on php7 due to the phpmock object dependencies, I'll fix that making sure to offer a compatible alternative for that too.